docs/questions/query-builder/editor.md

```diff
@@ -67,13 +67,9 @@ The data section is where you select the data you want to work with. Here you'll
 
 You can see the data source in a new browser tab by Cmd/Ctrl+Clicking on the data source's name in the query builder.
 
-To choose which columns to include in your query, click on the arrow next to the data source. You'll also be able [hide columns](../visualizations/table.md#rearranging-adding-and-removing-columns) from the table view once you visualize your results.
+To choose which columns to include in your query, click on the arrow next to the data source. If you uncheck a column, you'll still be able to use it while building a query (for example, in filters) but it won't be displayed in the results.
 
-### Adding or removing columns in a table
```

Comment:
Removing this as a section and moving the image to table viz article so as not to create an impression that this process affects the query